frontend/src/Movie/Details/MovieDetails.js

```diff
@@ -281,7 +281,7 @@ class MovieDetails extends Component {
       onMonitorTogglePress,
       onRefreshPress,
       onSearchPress,
-      queueDetails,
+      queueItems,
```

Comment:
What's purpose of changing this variable?

Reply:
MovieDetailsConnector has that answer :) And MovieStatusLabel has the answer to why that one was changed as well incase you ask on that file.